tests/aws_lambda_fsm/test_fsm.py

```diff
@@ -32,7 +32,7 @@
 
 class TestAction(Action):
     def execute(self, context, obj):
-        return 'ok'
+        return 'c'
```

Comment:
`ok` caused problems with the local loop.  `c` is fine because it is in the test stub